pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_managed_reconcile.go

```diff
@@ -178,18 +178,18 @@ func (r *managedReconcile) cleanUpAppliedManifestWorks(ctx context.Context, klus
 	}
 
 	var errs []error
-	for _, appliedManifestWork := range appliedManifestWorks.Items {
+	for index, appliedManifestWork := range appliedManifestWorks.Items {
```

Comment:
should this be `for index := range appliedManifestWorks.Items`?

Reply:
> should this be `for index := range appliedManifestWorks.Items`?

done